guides/doc-Provisioning_Guide/topics/Virt-Libvirt.adoc

```diff
@@ -161,16 +174,17 @@ To add KVM images on {ProjectServer}, complete the following steps:
 . From the *Architecture* list, select the operating system architecture.
 . In the *Username* field, enter the SSH user name for image access. This is normally the `root` user.
 . In the *Password* field, enter the SSH password for image access.
-. From the *User data* list, select if you want images to support user data input, such as `cloud-init` data.
 . In the *Image path* field, enter the full path that points to the image on the KVM server. For example:
 +
 [options="nowrap" subs="+quotes"]
 ----
- /var/lib/KVM/images/TestImage.qcow2
+ /var/lib/libvirt/images/TestImage.qcow2
 ----
 +
 . Click *Submit* to save the image details.
 
+NOTE: Libvirt Compute Resrouce does not allow seeding virtual instances via *User data*. When Finish template kind template is associated with the Operating System, {Project} will attempt to SSH using the provided *Username* and *Password* to the host and execute Finish template instead. 
+
```

Comment:
Again, @lzap, please be mindful of adding content rather than just marking up as part of this PR. 
Also, we don't mention in the docs things that aren't possible. We keep to only things that are possible, and then we imply that if it is not there, it is not possible.

Reply:
Going to split this.

Reply:
Going to split this.